store/sagas/contacts.sagas.js

```diff
@@ -92,7 +92,7 @@ export function* save({ domain, token, contactData }) {
     if (responseAction) {
       const response = responseAction.payload;
       const jsonData = yield call(() => new Promise((resolve) => {
-        resolve(response.json());
+        resolve(response.clone().json());
```

Comment:
The call to .json() is already returning a Promise, so there should be no need to resolve it and wrap it in a new Promise.  Instead this is possible, Line #28 should be:

const jsonData = yield response.json();

This will yield the Promise returned by the .json() invocation